CliClient/locales/de_DE.po

```diff
@@ -1704,8 +1728,9 @@ msgid "In: %s"
 msgstr "In: %s"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/NoteText.min.js:1661
+#, fuzzy
 msgid "Front"
-msgstr ""
+msgstr "Vorderseite"
```

Comment:
Why did you add `fuzzy` there? In Poedit this is the flag `Needs Works`.

This means that the translation won't be used.